test/model/channel_test.dart

```diff
@@ -56,6 +56,17 @@ void main() {
 
       await store.addSubscription(eg.subscription(stream1));
       checkUnified(store);
+
+      await store.handleEvent(eg.channelUpdateEvent(store.streams[stream1.streamId]!,
```

Comment:
If adding this to the `'added by events'` test, maybe good to rename the test to `'added/updated by events'`.